errbot/backends/slack.py

```diff
@@ -394,17 +417,21 @@ def _message_event_handler(self, event):
 
     def userid_to_username(self, id_):
         """Convert a Slack user ID to their user name"""
-        user = [user for user in self.sc.server.users if user.id == id_]
+        users = self.api_call('users.list')
```

Comment:
When trying to introspect `self.sc.server.users` I was unable to at all when testing. Using the `api_call('users.list')` I was able to. That was the main reason around it.

The groups appear to work with `sc.server.groups`. I can go back and leave that original call and see if that works.